src/inverseRobot.js

```diff
@@ -25,6 +25,19 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const value = Object.keys(robot);
+  const key = Object.values(robot);
+  const res = {};
+
+  for (let i = 0; i < value.length; i++) {
```

Comment:
do you really need index here? can't you use `for ... in` without creating additional variables?